openvdb_vendor/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+cmake_minimum_required(VERSION 3.18)
+project(openvdb_vendor)
+
+find_package(ament_cmake REQUIRED)
+find_package(ament_cmake_vendor_package REQUIRED)
+
+ament_vendor(${PROJECT_NAME}
+  VCS_URL https://github.com/AcademySoftwareFoundation/openvdb.git
+  VCS_VERSION v10.0.1
+  VCS_TYPE git
+  CMAKE_ARGS
```

Comment:
I'd add a bit more flags here
```cmake
diff --git a/openvdb_vendor/CMakeLists.txt b/openvdb_vendor/CMakeLists.txt
index da6d8ff..a058cc8 100644
--- a/openvdb_vendor/CMakeLists.txt
+++ b/openvdb_vendor/CMakeLists.txt
@@ -10,6 +10,10 @@ ament_vendor(${PROJECT_NAME}
   VCS_TYPE git
   CMAKE_ARGS
     -DCONCURRENT_MALLOC=Tbbmalloc
+    -DOPENVDB_CORE_SHARED=ON
+    -DOPENVDB_CORE_STATIC=OFF
+    -DOPENVDB_BUILD_BINARIES=OFF
+    -DUSE_CCACHE=ON
 )
 
 ament_export_include_directories("opt/${PROJECT_NAME}/include")
```

Just to make sure we use ccache when available and to avoid building stuff we don't consume (static libraries, binary with examples, visualizer, etc)

Reply:
I don't want to specify project defaults. The less delta the better I think.

- Shared is already on
- This package might not use statics, but this 'general' package shouldn't assume that from all consumers?
- Tools like ccache should be configured outside of projects. On a typical host and also on the buildfarm these are configured already.
- `OPENVDB_BUILD_BINARIES` might be put to off. As only the libraries are of interest, I'll add.

Reply:
Make sense. Thanks for the detailed reply